lib/l10n/app_en.arb

```diff
@@ -0,0 +1,29 @@
+{
+    "portfolio" : "Portfolio",
+    "moves": "Moves",
```

Comment:
Seria interessante colocar a descrição, para saber onde cada um é usado.
Exemplo: 
 "failureButton" : "Try again",
 "@failureButton": {
      "description": "button of default failure screen"
  }